docs/source/get_started/new_project.md

```diff
@@ -89,47 +92,65 @@ kedro run
 The first time you type a `kedro` command in your new project, you will be asked whether you wish to opt into [usage analytics](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-telemetry). Your decision is recorded in the `.telemetry` file so that subsequent calls to `kedro` in this project do not ask you again.
 ```
 
-When the command completes, you should see a log message similar to the following in your console:
+## Visualise your project
```

Comment:
When we add our style guide, let's make it clear that we use British spelling, since I often forget or get it wrong... 😅

Reply:
@astrojuanlu We have a style guide, and it mentions this. I also forget this at times, having recently been in a team that used US English 🤦 

https://docs.kedro.org/en/stable/contribution/documentation_contributor_guidelines.html#kedro-documentation-style-guide